src/build/webpack.config.test.js

```diff
@@ -85,18 +85,26 @@ testCompilation("first chapter of training material template", {
 
 
         <!-- .slide: class="page-questions" -->
-      `,
+      `.replaceAll(/^        /gm, ""),
```

Comment:
I'm not sure what this is for. I guess it removes indentation but I don't know why. Do you remember?

Reply:
Don't remember. Best is to try while removing it, I guess it will raise some issue (cannot test it right now).

Reply:
It's because indented blocks are compiled as code by the markdown compiler. But I don't understand why this wasn't a problem before 🤔 .

Reply:
Because of order of loaders I guess.

Reply:
Right! Before, title was extracted before markdown compilation.